base_jsonify/tests/test_get_parser.py

```diff
@@ -100,8 +100,8 @@ def test_json_export(self):
                     "email": None,
                 }
             ],
-            "create_date": fields.Datetime.to_string(partner.create_date),
-            "date": fields.Date.to_string(partner.date),
+            "create_date": partner.create_date.isoformat(),
```

Comment:
is there timezone information in there?

Reply:
yes there is. https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat

![image](https://user-images.githubusercontent.com/544090/67954978-1d13b900-fbf2-11e9-81d1-551d8df289a2.png)

Reply:
@sbidoul fixed